pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "redact"
-version = "10.0.1"
+version = "11.0.0"
```

Comment:
We discussed internally and we are fine with going to `10.1.0` it's technically not 100% correct but should be fine in this case. Retuning a Path instead of None should be a non breaking change. Only the exception is kind of a breaking change.